src/ocrd_network/server_utils.py

```diff
@@ -210,12 +212,12 @@ def validate_job_input(logger: Logger, processor_name: str, ocrd_tool: dict, job
         raise_http_exception(logger, status.HTTP_404_NOT_FOUND, message)
     try:
         report = ParameterValidator(ocrd_tool).validate(dict(job_input.parameters))
-        if not report.is_valid:
-            message = f"Failed to validate processing job input against the tool json of processor: {processor_name}\n"
-            raise_http_exception(logger, status.HTTP_400_BAD_REQUEST, message + report.errors)
     except Exception as error:
```

Comment:
When would this exception be raised? Validators should not raise errors but return a report. If you don't have a specific use case, it might be better to just not do `try/except` so that potential errors are actually raised and fixed.

Reply:
I wanted to be extra secure against unexpected errors. The workers or the server should never crash due to some error. The exceptions are still logged.

Reply:
Understood, but errors in

```
report = ParameterValidator(ocrd_tool).validate(dict(job_input.parameters))
```

would mean that something is broken in our implementation or in jsonschema. So this would likely not be a fluke that happens once, we would need to stop processing and fix the bug at the source. The only variable factor is the `job_input.parameters` dict and if user-provided input breaks the validator - which it must absolutely never do - we should fix that in the validator.

Reply:
> The only variable factor is the `job_input.parameters` dict and if user-provided input breaks the validator - which it must absolutely never do - we should fix that in the validator.

A failing validator would also probably fail most of the processing/workflow jobs. Input from the user breaking down the entire infrastructure is conceptually wrong to me. Even if all the processing jobs fail, the Processing Server should still respond to other requests such as checking logs of old jobs.  

There is also currently no graceful shutdown for the processing server. I.e., once the server dies, anything inside the internal processing cache of the server (not the RabbitMQ) will be lost.